e2e/__tests__/babelPluginJestDeadlines.test.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as path from 'path';
+import {skipSuiteOnJasmine} from '@jest/test-utils';
+import {extractSummary, runYarnInstall} from '../Utils';
+import runJest from '../runJest';
+
+const DIR = path.resolve(__dirname, '..', 'babel-plugin-jest-deadlines');
+
+beforeEach(() => {
```

Comment:
```suggestion
beforeAll(() => {
```

Reply:
Heh, this also exists in the test it's copied from. Not a problem there, as they only have one test.